### PR TITLE
i/6787: Allow checking `external/*/packages` dir for test files

### PR DESCRIPTION
**Pull request by panr**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Feature: Allowed `ckeditor5-dev-tests` package to search for test files in the `external/*/packages` directory. See ckeditor/ckeditor5#6787.

### CODE REVIEW

File: packages/ckeditor5-dev-tests/lib/utils/transformfileoptiontotestglob.js [hunk 1]
```diff
@@ -51,7 +63,7 @@ function transformSingleGlobPattern( globPattern, options ) {
 	const chunks = globPattern.split( globSep );
 	const packageName = chunks.shift();
 	const globSuffix = [ 'tests', '**' ];
-	let returnChunks = cwdChunks.concat( [ 'packages' ] );
+	let returnChunks = cwdChunks.concat( options.externalPackages ? [ 'external', '**', 'packages' ] : [ 'packages' ] );
```
Comment (Reinmar) [inline packages/ckeditor5-dev-tests/lib/utils/transformfileoptiontotestglob.js:66]:
```suggestion
	let returnChunks = cwdChunks.concat( options.externalPackages ? [ 'external', '*', 'packages' ] : [ 'packages' ] );
```

**Comment by pomek** [comment]
image

What is it?

**Comment by panr** [comment]
Huh, interesting. I'm on it...

**Comment by panr** [comment]
Just removed it, some strange things happened, but overall I think it's OK now 🤔

**Pull request merged**